osu.Game.Rulesets.Taiko.Tests/Skinning/TestSceneDrawableTaikoMascot.cs

```diff
@@ -175,25 +165,27 @@ private void setBeatmap(bool kiai = false)
             scoreProcessor.ApplyBeatmap(Beatmap.Value.Beatmap);
         }
 
-        private void createDrawableRuleset()
+        private void prepareTest(bool kiai)
```

Comment:
Can this be named something more like what it's doing? `prepareDrawableRuleset` or something. As it is, there are tests which aren't calling this method, and that appears wrong at the outset.

Reply:
Hmm, I intentionally made it more generic because it's basically doing full test setup (track etc.)